### PR TITLE
Add new tail logos

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

**Fixes:** #61 

## Summary:

Screen Shot 2021-02-02 at 9 22 31 PM

## Type of change:

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] add new tail logos 

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/data/logoIcons.json [hunk 1]
```diff
@@ -260,6 +260,78 @@
...
+    {
+      "name": "tail-DE",
+      "title": "Condor Airlines",
+      "desc": "Logo image for Condor Airlines",
+      "viewBox": "0 0 856 830"
+    },
+    {
+      "name": "tail-EI",
+      "title": "Aer Lingus Airlines",
+      "desc": "Logo image for Aer Lingus Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:272]: Please remove 'Airlines

File: src/data/logoIcons.json [hunk 2]
```diff
@@ -260,6 +260,78 @@
       "desc": "Logo image for SriLankan Airlines",
       "viewBox": "0 0 856 830"
     },
+    {
+      "name": "tail-DE",
+      "title": "Condor Airlines",
+      "desc": "Logo image for Condor Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:266]: Please remove Airlines

File: src/data/logoIcons.json [hunk 3]
```diff
@@ -260,6 +260,78 @@
...
+    {
+      "name": "tail-EI",
+      "title": "Aer Lingus Airlines",
+      "desc": "Logo image for Aer Lingus Airlines",
+      "viewBox": "0 0 856 830"
+    },
+    {
+      "name": "tail-EK",
+      "title": "Emirates Airlines",
+      "desc": "Logo image for Emirates Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:278]: Please change to 'Airline'

File: src/data/logoIcons.json [hunk 4]
```diff
@@ -260,6 +260,78 @@
...
+      "viewBox": "0 0 856 830"
+    },
+    {
+      "name": "tail-FI",
+      "title": "Iceland Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:283]: Please change to Icelandair

File: src/data/logoIcons.json [hunk 5]
```diff
@@ -260,6 +260,78 @@
...
+      "desc": "Logo image for Iceland Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:284]: Please change to Icelandair

File: src/data/logoIcons.json [hunk 6]
```diff
@@ -260,6 +260,78 @@
...
+    },
+    {
+      "name": "tail-KE",
+      "title": "Korean Airlines",
+      "desc": "Logo image for Korean Airlines",
+      "viewBox": "0 0 856 830"
+    },
+    {
+      "name": "tail-LA",
+      "title": "Latam Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:307]: LATAM

File: src/data/logoIcons.json [hunk 7]
```diff
@@ -260,6 +260,78 @@
...
+      "desc": "Logo image for Latam Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:308]: LATAM

File: src/data/logoIcons.json [hunk 8]
```diff
@@ -260,6 +260,78 @@
...
+      "viewBox": "0 0 856 830"
+    },
+    {
+      "name": "tail-LP",
+      "title": "Latam Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:313]: LATAM

File: src/data/logoIcons.json [hunk 9]
```diff
@@ -260,6 +260,78 @@
...
+      "desc": "Logo image for Latam Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:314]: LATAM Airlines Peru

File: src/data/logoIcons.json [hunk 10]
```diff
@@ -260,6 +260,78 @@
...
+      "viewBox": "0 0 856 830"
+    },
+    {
+      "name": "tail-LY",
+      "title": "EL AL Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:319]: El Al

File: src/data/logoIcons.json [hunk 11]
```diff
@@ -260,6 +260,78 @@
...
+      "desc": "Logo image for EL AL Airlines",
```
Comment (gusnaughton) [inline src/data/logoIcons.json:320]: El Al Israel Airlines

**Review by gusnaughton — CHANGES_REQUESTED** [review]

**Review by vidalmenAS — APPROVED** [review]

**Review by gusnaughton — APPROVED** [review]

**Pull request merged**